images/win/scripts/Installers/Install-AndroidSDK.ps1

```diff
@@ -0,0 +1,147 @@
+################################################################################
+##  File:  Install-AndroidSDK.ps1
+##  Desc:  Install and update Android SDK and tools
+################################################################################
+
+# install command-line tools
+$cmdlineToolsUrl = "https://dl.google.com/android/repository/commandlinetools-win-7302050_latest.zip"
```

Comment:
is it ok to hardcode the version? will we have to manually update it in future?

Reply:
We have never updated the version of SDK tools back in the days, so I'm not sure we need to change something here.